### DISCUSSION TITLE
error while implementing vpc-application loadbalancer (36th session)

### DISCUSSION THREAD

**Discussion by vyshuvenu** [learndevopsonline/learndevopsonline#1457] [Q&A]
my workstation pubilc-ip is 3.88.201.49
![Screenshot (48)](https://github.com/learndevopsonline/learndevopsonline/assets/139372307/81943195-ab42-43dc-bf1e-29ce0e7a6986)
before executing roboshop-terraform1, there is no loadbalancer existing in my aws account. but i still get this error. please tell me the reason sir @r-devops

**Comment by sreedharm07** [comment]
hi @vyshuvenu, 
Here load balancer is creating with dev-alb name. we want to create 2 lbs. both are creating with same name thats why its throwing error as dublicating the name. i seen in local.tf you already declared names for 2 lbs for ptivate and public  but you are not calling here. 
instead of var.env-alb -- give local.sg_name as you declare in locals


![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/988ccb19-8fd9-4400-9f7d-897cada673bc)

..
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/31cf76fe-5565-4ca5-9f96-b98c0b510820)

**Reply by vyshuvenu** [reply]
ok thanku for the quick response.
